Support for parametrized multilanguage testing

racertop opened the pull request:
**Description**

This PR adds a way to use parametrized tests for the Hedy translator. For this I added:

- A dictionary with the languages we support.
- A function to feed the keywords and languages to the tester.

**Fix for**

Fixes #1589 

**How to test**

Describe how to run your code to see it is action. See this https://github.com/Felienne/hedy/pull/880#issue-1016304308 for an example.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [ ] Links to an existing issue or discussion (if not, create an issue first)
- [ ] Describes changes clear in the format above (present tense, no subject)
- [ ] Has a "how to test" section
- [ ] Only one thing is done in this pull request (specifically please try to refrain from mixing textual changes to the yamls from code changes)

Felienne commented on tests/test_translation_level_02.py:
```diff
@@ -11,6 +13,7 @@
 
 class TestsTranslationLevel2(HedyTester):
     level = 2
+    all_kwords = hedy_translation.all_keywords_to_dict()
```
Thanks for this work @racertop it is a great way to test more extensively!!

I would just like to ask, can we make this var name `all_keywords`? It is just 2 more letters and it is a lot more clear! In general we try not to use abbreviations unless they really save letters and are wellknown (like AST for abstract syntax tree)

racertop replied:
Ok! Do you want me to change it in the places where I use `print_kword` or `if_kword` too?

Felienne replied:
Yeah that would be nicer!

racertop commented:
The test is failing because the translation Dutch translation differs in a keyword with the Spanish and English version. Particulary the command `add` in English and Spanish differs from `add_list` in Dutch. I assumed those dictionaries would all have the same key.

Felienne commented:
Ow good catch! I updated that recently in English but I forgot to update
the others. Can you make it all the same as en?

On Fri, 14 Jan 2022 at 17:27, Jesús Pelay ***@***.***> wrote:

> The test is failing because the translation Dutch translation differs in a
> keyword with the Spanish and English version. Particulary the command add
> in English and Spanish differs from add_list in Dutch. I assumed those
> dictionaries would all have the same key.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Felienne/hedy/pull/1681#issuecomment-1013271211>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAHVBJP47D6OCA32ZFL7MODUWBFGZANCNFSM5L4TGYMQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

racertop commented:
> Ow good catch! I updated that recently in English but I forgot to update the others. Can you make it all the same as en?

Done! I changed all occurences of kword to keyword as well. If there's anything else to change, I will fix that! 

Thank you for your time!
